core/src/sigverify_stage.rs

```diff
@@ -296,8 +321,20 @@ impl SigVerifyStage {
         verifier: &mut T,
         stats: &mut SigVerifierStats,
     ) -> Result<(), T::SendType> {
```

Comment:
similarly for this function, we should have metrics on how long these extra loops are taking as part of the stage metrics

Reply:
Will look

Reply:
done